js/my_first_tour.js

```diff
@@ -0,0 +1,120 @@
+// Define the tour!
+var tour = {
+  id: "hello-hopscotch",
+  steps: [
+    {
+      title: "Navigation improvements!",
+      content: "We've improved the navigation for the Docker documentation!<br /><br />This short tour takes less than a minute. It highlights the changes, then returns you to the current page. You'll learn about:<br /><ul><li>New top navigation</li><li>Left-hand navigation</li><li>Feedback links</li><li>In-page navigation</li></ul>",
+      target: "main-content",
+      placement: "top",
+      xOffset: "center",
+      yOffset: "300px",
+      onShow: function () {
+        // Hide the arrow on the first navigation bubble
+        $('.hopscotch-bubble-arrow-container').css('visibility', 'hidden');
+      }
+    },
+    {
+      title: "Top Navigation",
+      content: "Use the top navigation to discover the types of content available.",
+      target: "top-nav",
+      placement: "bottom",
+      arrowOffset: "center",
+      width: "570px",
+      onShow: function () {
+        // Show the arrow again
+        $('.hopscotch-bubble-arrow-container').css('visibility', 'visible');
+      }
+    },
+    {
+      title: "Guides",
+      content: "Use the <b>Guides</b> tab to learn how to install, configure, and manage Docker as a whole, or to view docs archives for previous Docker versions.",
+      target: "top-nav",
+      placement: "bottom",
+      width: "570px"
+    },
+    {
+      title: "Product Manuals",
+      content: "Use the <b>Product Manuals</b> tab to learn about a specific Docker product, such as Docker Cloud or UCP.",
+      target: "top-nav",
+      placement: "bottom",
+      width: "570px",
+      arrowOffset: "140px"
+    },
+    {
+      title: "Glossary",
+      content: "Use the <b>Glossary</b> tab to learn terminology specific to Docker.",
+      target: "top-nav",
+      placement: "bottom",
+      width: "570px",
+      arrowOffset: "280px"
+    },
+    {
+      title: "Reference",
+      content: "Use the <b>Reference</b> tab to go to Docker reference information, such as API, CLI, and commands topics.",
```

Comment:
I think "CLI" and "command" refer to the same thing, right? Other than that, these changes look good, @londoncalling.